src/main/java/racingcar/controller/GameController.java

```diff
@@ -0,0 +1,29 @@
+package racingcar.controller;
+
+import racingcar.controller.numericgenerator.RandomNumericGenerator;
+import racingcar.model.Cars;
+import racingcar.model.GameResult;
+import racingcar.model.TryCount;
+import racingcar.view.InputView;
+import racingcar.view.OutputView;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class GameController {
+
+    public void play() {
+        Cars cars = new Cars(new RandomNumericGenerator(), InputView.inputRacingCars());
+        TryCount tryCount = new TryCount(InputView.inputTryCount());
+        GameResult gameResult = new GameResult(cars.getNames());
+
+        int totalAttempts = tryCount.getValue();
+        for(int attempt = 0; attempt < totalAttempts; attempt++) {
+            cars.move();
+            gameResult.addResult(cars.getCarPositions());
```

Comment:
```suggestion
            gameResult.addResult(cars.move());
```

행동에 대한 결과를 반환하는 느낌으로 move 한번의 결과에 각 car들의 이름과 position을 모아서 반환하면 어떨까요?  

이렇게 되면 getCarPosition을 안해도 될것 같아보입니다.

그리고 GameResult의 일급컬렉션에서 winner를 선출하는것도 가능해질것 같아보입니다.